### DISCUSSION TITLE
Alexandria wishlist

### DISCUSSION THREAD

**Discussion by gaetbout** [keep-starknet-strange/quaireaux#103] [Q&A]
Hey Cairo developers 🚀, 

Let's have a conversation about what you would like to see in this library.
We are constantly striving to improve and expand our library to better serve the needs of our community. Your input is valuable to us, so we want to hear your ideas and suggestions!

We invite you to discuss your ideas here. 
We will review and filter the requests, aiming to integrate as issues that can alter be solved by anyone willing to dive into it!

Your feedback is crucial in shaping the future of Alexandria. 

We greatly appreciate your participation, and we will do our best to incorporate your needs and preferences into our ongoing development efforts.

Let's build something great together! 🦀

**Comment by rkdud007** [comment]
1. want bitshift related functions ( ex. fn bitshift_left(word: u128, num_bits: u128) ) 
2. want more type supported methods of integer (ex.pow( u8, u8 ) / pow ( u256, u256 ) )
